src/content/lodash/function/after/spec.js

```diff
@@ -0,0 +1,24 @@
+const fns = {
+  lodash: require('./lodash'),
+  plain: require('./vanilla'),
+}
+
+describe('before', () => {
```

Comment:
shouldn't it be `after`?

```suggestion
describe('after', () => {
```